tests/lit-tests/2280.ispc

```diff
@@ -0,0 +1,30 @@
+// RUN:     %{ispc} --target=host --nowrap --nostdlib -g -O0 -o %t.o %s -E
+// RUN:     %{ispc} --target=host --nowrap --nostdlib -g -O0 -o %t.o %s --onlycpp
+
+// RUN: not %{ispc} --target=host --nowrap --nostdlib -g -O0         %s                     2>&1 | FileCheck %s -check-prefix=CHECK_ERROR_0
+// RUN: not %{ispc} --target=host --nowrap --nostdlib -g -O0         %s                     2>&1 | FileCheck %s -check-prefix=CHECK_ERROR_0
```

Comment:
What's the different between these two runs?

Reply:
Forgot to remove